src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/css/app.css

```diff
@@ -62,3 +62,35 @@ a, .btn-link {
     .blazor-error-boundary::after {
         content: "An error has occurred."
     }
+
+#blazor-default-loading {
+    height: 90vh;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+}
+
+    #blazor-default-loading svg {
+        width: 120px;
+        height: 120px;
+        stroke-width: 8px;
+        display: block;
+    }
+
+    #blazor-default-loading circle {
+        fill: none;
+        stroke: #e0e0e0;
+        transform-origin: 50% 50%;
+        transform: rotate(-90deg);
+    }
+
+    #blazor-default-loading #blazor-default-loading-progress {
+        stroke: #1b6ec2;
+        transition: 0.05s stroke-dashoffset ease-in-out;
```

Comment:
The animation won't be seen if there's more resources being loaded per second. We can remove the duration of the transition, so it takes the default of 0s. That way, it won't matter how many resources are loaded per second.

Reply:
> The animation won't be seen if there's more resources being loaded per second 

So if we're loading `>20` resources / second, will it be equivalent to a transition of 0s? If so, I'm fine leaving it as `0.05s` if loading is fewer than 20 resources / second and `0s` if more than 20 resources / second.

Reply:
The WebAssembly app had around 200 total resources and took < 2 seconds to load. I think it would be appropriate to have a transition of 0s for the progress ring to be more accurate with the displayed percentage.

Reply:
Is there any difference between a `0s` transition and no transition property specified at all?

Reply:
Without a transition property specified, it defaults to 0s with no transition animation. I think I can get rid of the transition property altogether.